app/scripts/directives/blocks.js

```diff
@@ -111,7 +111,12 @@ angular.module('confRegistrationWebApp')
   .directive('dateQuestion', function () {
     return {
       templateUrl: 'views/blocks/dateQuestion.html',
-      restrict: 'E'
+      restrict: 'E',
+      controller: function($scope){
+        if(angular.isDefined($scope.answer) && !($scope.answer.value instanceof Date)){
+          $scope.answer.value = new Date($scope.answer.value);
```

Comment:
Okay.  Good, because I noticed it was setting my date to 1969 or something like that.